KhivaCsharp/KhivaCsharp/Interop/DLLNormalization.cs

```diff
@@ -0,0 +1,120 @@
+        public extern static void max_min_norm_in_place([In, Out] ref IntPtr tss, [In] ref double high, [In] ref double low, [In] ref double epsilon);
+
+        
+         /// <summary> Normalizes the given time series according to its maximum-minimum value and its mean. It follows the following
+         /// formulae:
+         /// \f[
+         /// \acute{x} = \frac{x - mean(x)}{max(x) - min(x)}.
+         /// \f]
+         ///</summary>
+         /// <param name="tss">Expects an input array whose dimension zero is the length of the time series (all the same) and dimension
+         /// one indicates the number of time series.
+         ///</param>
+         /// <param name="result">An array with the same dimensions as tss, whose values (time series in dimension 0) have been
+         /// normalized by substracting the mean from each number and dividing each number by \f$ max(x) - min(x)\f$, in the
+         /// time series.</param>
+        [DllImport(DLLLibrary.khivaPath, CallingConvention = CallingConvention.Cdecl)]
+        public extern static void mean_norm([In] ref IntPtr tss, [Out] out IntPtr result);
+
+        
+         /// <summary> Normalizes the given time series according to its maximum-minimum value and its mean. It follows the following
+         /// formulae:
+         /// \f[
+         /// \acute{x} = \frac{x - mean(x)}{max(x) - min(x)}.
+         /// \f]
+         ///</summary>
+         /// <param name="tss">Expects an input array whose dimension zero is the length of the time series (all the same) and dimension
+         /// one indicates the number of time series.</param>
+        [DllImport(DLLLibrary.khivaPath, CallingConvention = CallingConvention.Cdecl)]
+        public extern static void mean_norm_in_place([In, Out] ref IntPtr tss);
+
+        
+         /// <summary> Calculates a new set of times series with zero mean and standard deviation one.
+         ///</summary>
+         /// <param name="tss">Time series concatenated in a single row.</param>
+         /// <param name="epsilon">Minimum standard deviation to consider. It acts as a gatekeeper for
+         /// those time series that may be constant or near constant.</param>
+         /// <param name="result">KhivaArray with the same dimensions as tss where the time series have been
+         /// adjusted for zero mean and one as standard deviation.</param>
+        [DllImport(DLLLibrary.khivaPath, CallingConvention = CallingConvention.Cdecl)]
+        public extern static void znorm([In] ref IntPtr tss, [In] ref double epsilon, [Out] out IntPtr result);
+
+        
+         /// <summary> Adjusts the time series in the given input and performs z-norm
+         /// inplace (without allocating further memory).
+         ///</summary>
+         /// <param name="tss">Expects an input array whose dimension zero is the length of the time
+         /// series (all the same) and dimension one indicates the number of
```

Comment:
join this two lines:

         /// series (all the same) and dimension one indicates the number of
         /// time series.</param>